test/libyul/yulOptimizerTests/conditionalSimplifier/clear_after_if_break.yul

```diff
@@ -0,0 +1,17 @@
+{
+    let y := mload(0x20)
+    for {} and(y, 8) { pop(y) } {
+        if y { break }
+    }
+}
+// ====
+// step: conditionalSimplifier
+// ----
+// {
+//     let y := mload(0x20)
+//     for { } and(y, 8) { pop(y) }
+//     {
+//         if y { break }
+//         y := 0
```

Comment:
Won't this trip the fuzzer like https://github.com/ethereum/solidity/issues/7424#issue-493390791 because we "increase" execution step count by an additional step every iteration in the worst case?

Reply:
unless a subsequent optimization step is going to simplify the loop further...

Reply:
You are right! Then I have to fix this first...